internal/git/git.go

```diff
@@ -27,7 +29,13 @@ func (r *Repo) DefaultBranch() (string, error) {
 	ref, err := r.Git("symbolic-ref", "refs/remotes/origin/HEAD")
 	if err != nil {
 		logrus.WithError(err).Debug("failed to determine remote HEAD")
-		return "", errors.New("failed to determine remote HEAD (does your repo have a remote configured?)")
+		// this communicates with the remote, so we probably don't want to run
+		// it by default, but we helpfully suggest it to the user. :shrug:
+		logrus.Warn(
+			"Failed to determine repository default branch. " +
+				"Ensure you have a remote named origin and try running `git remote set-head --auto origin` to fix this.",
```

Comment:
when / how often would this happen?

Reply:
Shrug. It happened to me but I couldn't reproduce it after cloning. I think I was mucking around with multiple origins to test something so maybe that was it? I figured it wouldn't hurt to suggest this here.